<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>148</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="publish_path">
       <property name="text">
        <string>Publish Path : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="publish_name_entry"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Start Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="publish_frames_start_entry"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>End Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="publish_frames_end_entry"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="publish_chk_strip">
         <property name="text">
          <string>Strip Namespaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="publish_chk_world">
         <property name="text">
          <string>World Space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="publish_chk_whole">
         <property name="text">
          <string>Whole Frame Geo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="publish_chk_exportselected">
         <property name="text">
          <string>Export Selected</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="publish_chk_uv">
         <property name="text">
          <string>UV Write</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_publish">
       <property name="text">
        <string>Publish</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_publish</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>publish_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>publish_button()</slot>
 </slots>
</ui>
